packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -162,6 +162,15 @@ contract Reserve is IReserve, Ownable, Initializable, UsingRegistry, ReentrancyG
     return (uint256(tobinTaxCache.numerator), TOBIN_TAX_DENOMINATOR);
   }
 
+  /**
+   * @notice Returns the tobin tax.
+   * @return The tobin tax amount as a fraction.
+   */
+  function getTobinTax() external view returns (uint256, uint256)
+  {
```

Comment:
The `{` should go on the previous line

Reply:
Thanks, realized I had to reinstall solhint